src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -35,7 +71,10 @@
 import com.cloudbees.hudson.plugins.folder.computed.FolderComputation;
 import com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger;
 import com.cloudbees.hudson.plugins.folder.views.AbstractFolderViewHolder;
+import com.google.common.collect.ImmutableSet;
```

Comment:
My IDE alphabetized imports.

`com.google.common.collect.ImmutableSet` is an addition.

Reply:
Avoid Guava like the plague. This can be replaced with JDK classes